inst/extdata/socioeconomics/Maddison_population.csv

```diff
@@ -1 +1 @@
Peru,4000,1300,1300,1317,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,2001,,,,,,,,,,,,,,,,,,,,2606,,,,,,,,,,,,,,,,,,,,3346,,,,,,3470,3513,3558,3603,3648,3694,3741,3788,3836,3885,3934,3984,4034,4085,4137,4189,4242,4295,4350,4405,4460,4517,4574,4631,4690,4764,4838,4914,4992,5070,5150,5230,5312,5396,5480,5569,5660,5752,5846,5941,6038,6137,6237,6338,6440,6550,6662,6776,6892,7010,7130,7252,7376,7502,7632.5,7826.2,8025.7,8232.1,8447,8671.5,8904.9,9146.1,9396.7,9657.8,9931,10217.5,10516.5,10825.8,11143.5,11467.3,11796.4,12132.2,12476,12829.1,13192.8,13568.3,13954.7,14350.3,14753.1,15161.199,15573.199,15990.099,16414.399,16848.699,17295.298,17754.798,18125.129,18516.941,18927.692,19348.926,19771.935,20195.924,20625.082,21063.8,21511.443,21967.278,22430.449,22900.004,23374.974,23846.388,24304.899,24748.122,25174.087,25581.874,25979.722,26376.033,26769.436,27158.869,27544.305,27925.628,28302.603,28674.757,29041.593,,35707.142
Uruguay,,,,55,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,132,,,,,,,,,,,,,,,,,,,,343,354,364,376,387,399,411,424,437,450,464,482,502,522,543,564,587,610,635,660,686,706,727,748,770,792,815,839,864,889,915,930,945,961,977,993,1009,1026,1043,1062,1081,1112,1144,1177,1223,1246,1269,1292,1316,1341,1371,1402,1433,1465,1498,1534,1571,1608,1646,1685,1713,1741,1770,1799,1829,1859,1889,1921,1952,1944,1965,1987,2010,2032,2055,2081,2107,2134,2160,2188,2194.275,2223.06,2252.965,2283.92,2316.543,2353.136,2389.29,2424.959,2460.411,2495.022,2530.969,2563.794,2598.466,2632.111,2663.61,2693.133,2721.207,2748.579,2776.607,2801.859,2823.659,2826.042,2829.526,2834.115,2838.213,2841.818,2857.453,2873.52,2889.495,2905.131,2919.851,2935.649,2953.997,2972.57,2990.097,3008.27,3026.971,3045.153,3063.668,3084.065,3105.708,3127.574,3149.262,3170.958,3193.413,3215.423,3238.28,3262.838,3284.853,3304.234,3323.876,3343.797,3363.085,3381.606,3399.237,3415.92,3431.932,3447.496,3462.531,,3721.919
Venezuela,,,,718,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,1324,,,,,,,,,,,,,,,,,,,,1653,1675,1699,1725,1753,1784,1816,1849
```

Comment:
(Or at least I don't think I did. I added the file, title, etc., lines on my local repository, but since it turned out that this file was being processed elsewhere, I didn't think I committed the changes to the csv file. If I did, I didn't mean to!)